bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -196,7 +198,7 @@ template_ids = cuts.apply_template_cuts(
     template_ids=template_ids)
 
 logging.info("%d out of %d templates kept after applying template cuts",
-             len(template_ids), len(trigs.singles[0].bank))
+             len(template_ids), original_bank_len)
```

Comment:
This fixes a small bug in the logging, where the original bank length was not correct